### PR TITLE
feat: Send mail to user when order successfully

### PR DESCRIPTION
**Pull request by KhoaAnhDoNguyen**

### CODE REVIEW

File: BackEnd/app/Http/Controllers/OrderController.php [hunk 1]
```diff
@@ -67,4 +72,45 @@ public function destroy($id)
...
+            \Log::error('Failed to send order confirmation email: ' . $e->getMessage());
+            return response()->json(['message' => 'Failed to send order confirmation email.'], 500);
+        }
+    }
+    
+
+    
+
+
+}
```
Comment (namdnhn) [inline BackEnd/app/Http/Controllers/OrderController.php]: thiếu space

File: BackEnd/routes/api.php [hunk 2]
```diff
@@ -48,9 +50,11 @@
...
 Route::middleware(['auth:sanctum', 'is_admin'])->group(function () {
-    Route::get('show-list-product', [ProductController::class, 'list']);
+  Route::get('show-list-product', [ProductController::class, 'list']);
 });
 Route::get('get-product/{id}', [ProductController::class, 'get_product_information']);
 
 Route::post('create-new-product', [ProductController::class, 'create_new_product']);
 Route::post('update-product/{id}', [ProductController::class, 'update_product']);
+
+Route::post('/send-order-confirmation', [OrderController::class, 'sendOrderConfirmation']);
```
Comment (namdnhn) [inline BackEnd/routes/api.php]: thiếu space

**Review by letanhoaUIT — APPROVED** [review]
ok nha

**Review by namdnhn — APPROVED** [review]

**Comment by KhoaAnhDoNguyen** [comment]
Ready

**Review by letanhoaUIT — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**